src/wifi/model/mac-low.cc

```diff
@@ -717,6 +717,20 @@ MacLow::StoredCurrentAllocation (void) const
   return m_allocationStored;
 }
 
+void
+MacLow::SetAmpduAckType (uint8_t numOfMpdus, const WifiMacHeader &hdr, MacLowTransmissionParameters &txParams) const
+{
+  NS_LOG_FUNCTION (numOfMpdus << txParams << hdr);
```

Comment:
since it doesn't need any member variable from MacLow it's ok to not log `this`, but in that case make the function static